find_large/files/core.py

```
@@ -122,11 +117,13 @@ def find_files(
 
     if output_file:
         try:
-            file_console: Console = formatting.Console(file=open(output_file, "w"), force_terminal=True)
+            file_console: Console = formatting.Console(
+                file=open(output_file, "w"), force_terminal=True
+            )
             formatting.format_table(data_lines, no_size, total_bytes, file_console, no_table)
             file_console.file.close()
             formatting.print_success(f"Results saved to {output_file}")
         except Exception as e:
```

Ruff (0.14.13)

Do not catch blind exception: `Exception`

(BLE001)